### PR TITLE
Question Scene MVP

### PR DESCRIPTION
**Pull request by tonyseetonydo**
Implemented the basic interface of the question scene. Right now we have one question and its options. When the user clicks on an option, it will be shown in the terminal as a print message.

TODOs:
- [ ] add click visual effect (change to a brighter/dimmer color)
- [ ] verify solution
- [ ] update player state and current scene state vars
- [ ] maybe add a next_question method?
- [ ] boxes' borders

### CODE REVIEW

File: src/client.py [hunk 1]
```diff
@@ -12,7 +12,8 @@
         SceneState.PLAYER_QUESTION: QuestionScene(screen, PlayerState(network), network),
         SceneState.QUIT: QuitScene(screen, PlayerState(network), network)
     }
-    scene = SceneState.ENTRY
+    # scene = SceneState.ENTRY
```
Comment (nickbar01234) [inline src/client.py:15]: Maybe we should still keep `SceneState.ENTRY` here?

File: src/modules/scene/question_scene.py [hunk 2]
```diff
@@ -19,22 +19,92 @@ def __init__(self, screen, player_state, network):
...
+                             box_colors[i % len(box_colors)], box)
+                # TODO: center text
+                text_surface = STYLE["font"]["text"].render(
+                    option, True, (0, 0, 0))
+                self.get_screen().blit(text_surface, box.center)
+
+            pg.display.flip()
+
+    def __create_option_boxes(self, options):
+        option_boxes, optons_box_borders = [], []
```
Comment (nickbar01234) [inline src/modules/scene/question_scene.py]: I don't think we ever used `options_box_borders` in the main loop.

Comment (tonyseetonydo) [reply src/modules/scene/question_scene.py]: Yeah, it's currently a placeholder. I have it as one of my TODOs

**Review by nickbar01234 — APPROVED** [review]
LGTM. I will merge the code to revamp the styling!

**Pull request merged**